guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -73,103 +73,195 @@ If you lose connection to the command shell where the upgrade command is running
-+
-* Ensure no Red Hat repositories are enabled by entering the command:
+Complete the following steps on the connected {ProjectServer}.
+
+. Ensure that you have synchronized the following repositories in your connected {ProjectServer}.
 +
+[options="nowrap" subs="attributes"]
 ----
-# yum repolist
+{RepoRHEL7ServerAnsible}
+{RepoRHEL7Server}
+{RepoRHEL7ServerSatelliteServerProductVersion}
+{RepoRHEL7ServerSatelliteMaintenanceProductVersion}
+{RepoRHEL7ServerSoftwareCollections}
 ----
-
-.Updating Disconnected {ProjectServer} to the Next Minor Version
-
-. Create a new configuration file as follows:
++
+. Download the debug certificate of the organization and store it locally at, for example, `/etc/pki/katello/certs/org-debug-cert.pem` or a location of your choosing.
+. Create a Yum configuration file under `/etc/yum.repos.d` with the following repository information:
 +
 [options="nowrap" subs="attributes"]
 ----
-# vi /etc/yum.repos.d/redhat-local.repo
-
 [{RepoRHEL7ServerAnsible}]
 name=Ansible {SatelliteAnsibleVersion} RPMs for Red Hat Enterprise Linux 7 Server x86_64
-baseurl=file:///var/lib/pulp/published/yum/https/repos/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/ansible/2.9/os/
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel/server/7/$releasever/$basearch/ansible/{SatelliteAnsibleVersion}/os/
 enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
 
 [{RepoRHEL7Server}]
 name=Red Hat Enterprise Linux 7 Server RPMs x86_64
-baseurl=file:///var/lib/pulp/published/yum/https/repos/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/os/
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/os/
 enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
 
 [{RepoRHEL7ServerSatelliteServerProductVersion}]
 name={ProjectNameX} for RHEL 7 Server RPMs x86_64
-baseurl=file:///var/lib/pulp/published/yum/https/repos/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/satellite/{ProjectVersion}/os/
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/satellite/6.10/os/
 enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
 
 [{RepoRHEL7ServerSatelliteMaintenanceProductVersion}]
 name={ProjectName} Maintenance 6 for RHEL 7 Server RPMs x86_64
```

Comment:
This is OK for a cherry pick to an older version but heads up that this will change for Satellite 7 and technically master should already be following 7.